addtocart/addtocart.css

```diff
@@ -0,0 +1,233 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: 'Poppins', sans-serif;
+    --product-name: #ff2020;
+    --primary-color: #e72744;
+    --secondary-color: #F05524;
+    --primary-btn: #6495ED;
+}
+/* navbar */
+.landing-page-nav{
+  display:flex;
+  justify-content: space-between;
+  background-color: #f1f1f1;
+  padding: 10px;
+  margin-bottom: 20px;
+}
+.nav-links {
+    padding-top: 20px;
+}
+
+.nav-links a {
+    color: black;
+    text-decoration: none;
+    padding-right: 15px;
+}
+
+.brand-name{
+  color:var(--primary-color);
+  font-size: 2.5rem;
+  font-weight: bold;
+}
+.brand-sub-title{
+  color:var(--secondary-color);
+   margin-top:-5px;
+    margin-left: 1rem;
+  
+}
+.search-bar{
+  max-width:800px;
+  width:100%;
+  
+}
+.form{
+  display:flex;
+  
+}
+
+.search-bar-input{
+  flex-grow: 1;
+  padding:16px;
+}
+
+.submit-btn{
+  padding:15px;
+}
+.nav-icons{
+  display: flex;
+  justify-content: flex-end;
+  gap:1.2rem;
+  
+}
+.nav-icons a{
+  font-size:24px;
+  text-decoration: none;
+  margin-top:10px;
+  color:black;}
+/* cart container */
+.cart{
+  font-size:1.5rem;
+  font-weight:bold;
+  text-align:center;
+}
+
+.cart-container{
+  margin-top:3rem;
+  position:relative;
+}
+
+.imganddetails{
+  display:flex;
+  width:800px;
+  height:280px;
+  margin-left:8rem;
+  margin-top: 20px;
+  margin-bottom:20px;
+  border:1px solid black;
+  
+}
+.cart-product-image img{
+  height:100%;
+  padding:5px;
+}
+.product-details{
+   padding:5px 18px;
+}
+.product-details span{
+  display:inline-block;
+   padding:5px 18px;
+  
+  
+}
+.product-details .product-name{
+  font-weight: bold;
+  color:var(--primary-color)
+}
+.product-details .product-quantity{
+  border:1px solid black;
+  width:15%;
+  text-align: center;
+   margin-left: 12px;
+}
+.wishlist-and-cart{
+  display:flex;
+}
+.move-to-wishlist {
+    width: 90%;
+    padding: 0.5rem;
+    background-color: #eb2543;
+    color: white;
+    border: none;
+    display: inline;
+    margin:0.8rem;
+}
+.remove-from-cart {
+    width: 90%;
+    padding: 0.5rem;
+    background-color: grey;
+    color: white;
+    border: none;
+    display: inline;
+    margin:0.8rem;
+}
+
+.price-details{
+  position:absolute;
+  margin-right:220px;
+  top:0;
+  right:0;
+  
+}
+.price-details div{
+  padding-bottom: 10px;
+}
+.item-price,.discount,.delivery-charges,.total-amount{
+  display: flex;
+  justify-content: space-between;
+}
+.final-details{
+  margin-left:8rem;
+  
+}
+hr{
```

Comment:
Maybe create a class for this.

Reply:
thank you mahesh for valuable review.will  refactor all  this once i finish my  assignment to submit